private/headsurface.m

```diff
@@ -195,6 +195,10 @@
     otherwise
       ft_error('other surfaces cannot be constructed this way');
   end
+
+  %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+else
+    ft_error('headmodel or headshape required')
```

Comment:
it is a good idea to explicitly check that any of the if-elseif-elseif clauses was used.